osgar/drivers/oak_camera.py

```diff
@@ -318,8 +329,11 @@ def run_input(self):
 
         if self.oak_config_model is not None:
             # configure OAK neural network
-            assert self.is_color, "RGC camera must be enabled!"
-            self.config_oak_nn(pipeline, color)  # note, that "color" is RGB camera
+            if self.is_color:
+                self.config_oak_nn(pipeline, color)  # note, that "color" is RGB camera
+            else:
```

Comment:
What did I want to detect from a mono camera and use an RGB camera at the same time?